File: Happy_Number/C++/alsuga/happy_number_test.cc
#include <iostream>
#include <vector>
#include <cassert>
#include "happy_number.hpp"

using namespace std;

int main() {
  vector<int> happy = {1, 7, 10, 13, 19, 23, 28, 31, 32, 44, 49, 68, 70};
  for(int i = 0; i < happy.size(); ++i) {
    assert(is_happy_number(happy[i]) == true);
  }
  cout << "All test passed" << endl;
  return 0;
}

File: Happy_Number/Java/darubramha89/HappyNumber_test.java
package happy.number;

public class HappyNumber_test {

	public static void main(String[] args) {

		HappyNumber hn = new HappyNumber();
		for (int i = 0; i <= 1000; i++) {
			
			if(hn.isHappyNumber(i))
				System.out.println(i + " is a Happy Number :) ");
			else
				System.out.println(i + " is not a Happy Number :( ");
		}
	}
}
